packages/zundo/__tests__/options.test.ts

```diff
@@ -425,7 +425,7 @@ describe('Middleware options', () => {
       expect(storeWithHandleSet.temporal.getState().futureStates.length).toBe(
         2,
       );
-      expect(console.log).toHaveBeenCalledTimes(2);
+      expect(console.warn).toHaveBeenCalledTimes(2);
```

Comment:
to allow me to use console.log in zundo source code while testing and not mess up tests